src/common/components/ConnectWalletButton/ConnectWalletModal/ConnectWalletModal.tsx

```diff
@@ -0,0 +1,40 @@
+import { LinkIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+  Box,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+} from '@chakra-ui/react';
+
+import { RoundIcon } from '@/common/components/IconBox/RoundIcon';
+
+interface ConnectWalletModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const icon = <LinkIcon h={{ base: '8', md: '10' }} w={{ base: '8', md: '10' }} />;
+
+export const ConnectWalletModal = ({ isOpen, onClose }: ConnectWalletModalProps) => {
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        <Box left="50%" position="absolute" top="0" transform="translate(-50%, -50%)">
+          <RoundIcon icon={icon} />
```

Comment:
I don't think this is needed actually, there are other modals with no icon on top.
![image](https://user-images.githubusercontent.com/19740800/210308546-eb1ab2fd-e5f5-45c9-a716-1d06111648bd.png)

but let's wait for Rob to confirm

Reply:
removed it for now. if Rob confirms I will add it back.